deploy/FlapperInit.sol

```diff
@@ -148,29 +133,39 @@ library FlapperInit {
     }
 
     function initSplitter(        
-        DssInstance memory dss,
-        address splitter,
-        SplitterConfig memory cfg
+        DssInstance      memory dss,
+        SplitterInstance memory splitterInstance,
+        SplitterConfig   memory cfg
     ) internal {
+        SplitterLike    splitter = SplitterLike(splitterInstance.splitter);
+        SplitterMomLike mom      = SplitterMomLike(splitterInstance.mom);
+
         // Sanity checks
-        SplitterLike splitter_ = SplitterLike(splitter);
```

Comment:
And we are actually doing the reverse here also generating more diffs. So I think one way or the other it would be better to standardize and will reduce the amount of changes.